grammars/shell-unix-bash.cson

```diff
@@ -21,7 +21,7 @@
   'cygport'
   'bats'
 ]
-'firstLineMatch': '^#!.*\\b(bash|zsh|sh|tcsh|ksh|dash|ash|csh)|^#\\s*-\\*-[^*]*mode:\\s*shell-script[^*]*-\\*-'
+'firstLineMatch': '^#!.*\\b(bash|zsh|sh|tcsh|ksh|dash|ash|csh|rc)|(?i)(?:^\\s*#\\s*-\\*-[^*]*mode:\\s*shell-script[^*]*-\\*-)'
```

Comment:
Yep, confirmed. It does.

@50Wliu I've amended the expression to use terser syntax.